metadata-manager/README.md

```diff
@@ -64,7 +64,7 @@ on the model of the record.
 The metadata loader currently supports two Lambda functions: one for syncing from a tracking sheet and another for
 syncing from a custom CSV presigned URL. Upon a CREATE or UPDATE operation in the metadata library, the Lambda function
 will publish events to the `MainOrcabusEventBus` using the schema defined
-in [MetadataStateChange.schema.json](/docs/schemas/events/metadatamanager/MetadataStateChange.schema.json).
+in [MetadataStateChange.schema.json](./app/schema/events/MetadataStateChange/MetadataStateChange.schema.json).
```

Comment:
Any particular reason for the move from `docs/` ?

Reply:
I usually open the metadata-manager app directly in PyCharm from the app folder, so the schema feels more like a part of the app rather than something at the root level. Well I could place the schema in /docs and generating the Python models from it within the app, which kinda align with old repo anyway (as schema is deployed from different stack)